app/src/main/res/drawable/ic_cognitive.xml

```diff
@@ -0,0 +1,22 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Good choice on using the vector asset.  Was this a direct export from Figma?

Reply:
I exported the svg from figma and then imported to studio which transforms svg to the vector.